### PR TITLE
Support/wagtail 52

### PR DESCRIPTION
**Pull request by katdom13**
## Includes the following PR:
[Wagtail 5.1 PR](https://github.com/torchbox/wagtailguide/pull/27)

## [Wagtail 5.2 release notes](https://docs.wagtail.org/en/stable/releases/5.1.html)

Changes:
- [Adoption of classname convention for MenuItem related classes and hooks](https://docs.wagtail.org/en/stable/releases/5.2.html#adoption-of-classname-convention-for-menuitem-related-classes-and-hooks)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -26,5 +26,6 @@
     include_package_data=True,
     install_requires=[
         "wagtail>=4.1",
+        "Django>=3.2",
```
Comment (nickmoreton) [inline setup.py:29]: Seems like a good idea to me.

File: .github/workflows/main.yml
Comment (nickmoreton) [inline .github/workflows/main.yml:1]: I think testing of django 4.1 is now missing here.

File: wagtail_guide/tests/settings.py [hunk 2]
```diff
@@ -79,3 +79,5 @@
 }
 
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
+
+STATIC_URL = "/static/"
```
Comment (nickmoreton) [inline wagtail_guide/tests/settings.py:83]: I'm only wondering why this was added?

**Review by nickmoreton — CHANGES_REQUESTED** [review]

File: wagtail_guide/tests/settings.py (hunk 2 above)
Comment (katdom13) [reply wagtail_guide/tests/settings.py:83]: Hi @nickmoreton ,
So this error can be avoided:
https://github.com/torchbox-forks/wagtailguide/actions/runs/6809761204/job/18516669914#step:5:76

File: .github/workflows/main.yml
Comment (katdom13) [reply .github/workflows/main.yml:1]: This commit lists out all the possible tests:
https://github.com/torchbox-forks/wagtailguide/pull/3/commits/981963595018e93d49de4232a1662e7cad14dcb7

Thanks!

**Review by nickmoreton — APPROVED** [review]
Hi @katdom13 Looks good I approve.

**Pull request merged**